src/testdir/test_i_ctrl_rsb.vim

```diff
@@ -0,0 +1,24 @@
+" Test for InsertCharPre autocmd
+
```

Comment:
This comment doesn't match the file name

Reply:
Or, maybe, these functions should just be in `test_edit.vim` rather than a new file, and should follow the convention (e.g. 2-space indent) of existing test functions.